### ISSUE TITLE
Fix request: extend ImplementEqualsHashCodeOnValueObjects for Lombok annotations

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#135]
Classes with only Lombok @Getter and no @EqualsAndHashCode (or @Value or @Data) should trigger a violation as well.
Example:
```
import lombok.*;

@Getter
class LombokGetterBad { // bad - equals and hashCode missing
    private String someState1 = "some1";
    private String someState2 = "some2";
}

@Getter
@EqualsAndHashCode
class LombokGetterGood { // good  
    private String someState1 = "some1";
    private String someState2 = "some2";
}
```

**Comment by jborgers** [comment]
updated documentation page too, with Lombok as first solution, and AutoValue and Immutables framework added as well.
